index.js

```diff
@@ -10,14 +10,14 @@ const typeMappings = {
 };
 
 function checkType(type) {
-	if (type in typeMappings) {
+	if (Object.hasOwnProperty.call(typeMappings, type)) {
```

Comment:
Was it intentional to look down the prototype chain?

I would expect this to be an invalid type error: `await locatePath(files, 'toString');`
<img width="378" alt="image" src="https://user-images.githubusercontent.com/32409546/172062096-527f8feb-0a66-46f9-9db3-dd95a77c66c3.png">

`typeMappings.hasOwnProperty` may be more appropriate:
<img width="396" alt="image" src="https://user-images.githubusercontent.com/32409546/172062209-74d39ddb-8bfd-4309-8a8c-0eab0ab56609.png">